src/qt/transactiontablemodel.cpp

```diff
@@ -95,25 +95,27 @@ class TransactionTablePriv
      */
     QList<TransactionRecord> cachedWallet;
 
-    bool fQueueNotifications = false;
+    bool m_loaded = false;
+    int m_progress = 100;
     std::vector< TransactionNotification > vQueueNotifications;
 
     void NotifyTransactionChanged(const uint256 &hash, ChangeType status);
-    void ShowProgress(const std::string &title, int nProgress);
+    void DispatchNotifications();
 
     /* Query entire wallet anew from core.
      */
     void refreshWallet(interfaces::Wallet& wallet)
     {
-        qDebug() << "TransactionTablePriv::refreshWallet";
-        cachedWallet.clear();
+        assert(cachedWallet.empty());
```

Comment:
With new logic I'd prefer to `assert(!m_loaded);` or 
```suggestion
        assert(cachedWallet.empty() && !m_loaded);
```